wgpu-hal/examples/ray-traced-triangle/main.rs

```diff
@@ -284,9 +284,9 @@ impl<A: hal::Api> Example<A> {
         dbg!(&surface_caps.formats);
         let surface_format = if surface_caps
             .formats
-            .contains(&wgt::TextureFormat::Rgba8Snorm)
+            .contains(&wgt::TextureFormat::Bgra8Unorm)
```

Comment:
What's going on here?

Reply:
That was a modification because dx12 didn't support that format, I can revert it but then I'm unable to use that example on directx12

Reply:
Since this is just the HAL example I can probably remove it if wanted (I started work on this before ray-tracing in core was merged).

Reply:
Oh it was Snorm... what. Everyone should support `Rgba8Unorm`